tests/pos/i7264b.scala

```diff
@@ -0,0 +1,7 @@
+import scala.quoted._
+class Foo {
+  def f[T2: Type](e: Expr[T2])(given QuoteContext) = e match {
+    case '{ $x: *:[Int, $t] } =>
```

Comment:
@odersky  unfortunately this case still has the same issue. Note that this time it is in a type within quoted Expr pattern.